roles/apm_agent/defaults/main.yml

```diff
@@ -1,13 +1,15 @@
 ---
-cd_app_service: fluent-bit
+cd_app_service: fluent-bit{{ env_path_suffix }}
 
 apm_agent:
   name: "Fluent Bit"
   version: "1.7.4"
   url:
     zip: "http://bwa.nrs.gov.bc.ca/int/artifactory/ext-binaries-local/fluent/fluent-bit/1.7.4/fluent-bit.tar.gz"
   install_dir: "/apps_ux/fluent-bit"
-  service_dir: "/apps_ux/s6_services/fluent-bit"
+  service_dir: "/apps_ux/s6_services/fluent-bit{{ env_path_suffix }}"
 
 apm_agent_home: "/apps_ux/agents/main"
+
+env_path_suffix: ""
```

Comment:
Is env_path_suffix mandatory?  What about logic or enforcement for hyphenation?  E.g. fluent-bit-things vs fluent-bitthings.

Reply:
It's not mandatory. At this point there's nothing to enforce hyphenation. I'm setting the value (e.g. "-int", "-dlvr") in an Ansible inventory file.